### PR TITLE
Add instructions and functionality for testing new release notes

### PR DESCRIPTION
**Pull request by kenzieschmoll**
@johnpryan

### CODE REVIEW

**Comment by johnpryan** [comment]
LGTM

File: packages/devtools_app/lib/src/shared/release_notes/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+## Writing DevTools release notes
+Release notes for DevTools are hosted on the flutter website (see [archive](https://docs.flutter.dev/development/tools/devtools/release-notes)).
+To add release notes for the latest release, create a PR with the appropriate changes for your release (see example
+[PR](https://github.com/flutter/website/pull/6791)).
+
+Test these changes locally before creating the PR. See [README.md]() for getting setup to run the Flutter website locally.
```
Comment (elliette) [inline packages/devtools_app/lib/src/shared/release_notes/README.md:6]: README link missing?

File: packages/devtools_app/lib/src/shared/release_notes/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+One you are satisfied with the release notes, stage the flutter website on firebase and point the DevTools release notes
```
Comment (elliette) [inline packages/devtools_app/lib/src/shared/release_notes/README.md:8]: Nit: capitalize Flutter

File: packages/devtools_app/lib/src/shared/release_notes/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+logic to this staged url.
+
+### Staging your changes on firebase
```
Comment (elliette) [inline packages/devtools_app/lib/src/shared/release_notes/README.md:11]: Also Firebase

**Review by elliette — APPROVED** [review]

**Pull request merged**